app/controllers/frontend_controller.rb

```diff
@@ -174,15 +174,15 @@ def set_display_currency
   end
 
   def quickstarts_presenter
-    return if @quickstarts_presenter
-
-    current_api = if @backend_api&.id
-                    @backend_api
-                  elsif @service&.id
-                    @service
-                  else
-                    current_account.default_service
-                  end
-    @quickstarts_presenter = QuickstartsPresenter.new(current_api)
+    @quickstarts_presenter ||= begin
+      current_api = if @backend_api&.id
+                      @backend_api
+                    elsif @service&.id
+                      @service
+                    else
+                      current_account.default_service
+                    end
+      QuickstartsPresenter.new(current_api)
+    end
```

Comment:
why do we need the nesting? Why not set `current_api` as before and then set `@quickstarts_presenter`?

Reply:
no need, the alternative would be:

```
return @quickstarts_presenter if @quickstarts_presenter
```

and then as it was before

I just thought that it was repeating `@quickstarts_presenter` too many times :grimacing:

Reply:
Ok. I'm personally not in favor of updating code that works just for formatting or minor improvements (because each change makes tracking the origins of the code harder) but if it bothered you, that's fine

Reply:
Well, the code was broken, as it was returning a `nil` the second time `quickstarts_presenter` was accessed.
And while this needed a fix anyway, I decided to make it nicer-looking (in my opinion, obviously)